src/protagonist/API/Features/Image/AssetBeforeProcessing.cs

```diff
@@ -4,15 +4,15 @@ namespace API.Features.Image;
 
 public class AssetBeforeProcessing
 {
-    public AssetBeforeProcessing(Asset asset, DeliveryChannelsBeforeProcessing[] deliveryChannelsBeforeProcessing)
+    public AssetBeforeProcessing(Asset asset, DeliveryChannelsBeforeProcessing[]? deliveryChannelsBeforeProcessing)
     {
         Asset = asset;
         DeliveryChannelsBeforeProcessing = deliveryChannelsBeforeProcessing;
     }
 
     public Asset Asset { get; }
 
-    public DeliveryChannelsBeforeProcessing[] DeliveryChannelsBeforeProcessing { get; }
+    public DeliveryChannelsBeforeProcessing[]? DeliveryChannelsBeforeProcessing { get; set; }
```

Comment:
If we don't set this to data loaded from DB can it go back to being immutable? This makes it easier to reason about as it's set once then passed down